### ISSUE TITLE
Issue where you have 2 modals on the same page doing different things.

### ISSUE THREAD

**Issue by jordanbrown** [codrops/ModalWindowEffects#5]
Has anyone come up on the issue where you have 2 modals with 2 different modal ids conflicting? They both work fine but the click on the overlay doesnt close the modal that is second in the layout.

**Comment by so-ale** [comment]
Can you be more specific? Maybe give us an example?

I maybe facing a similar issue where I have two anchor links with data-modal attributes linked to two different ids, to open those modals. However, only the first link triggers open the modal. Second one doesn't. 

Is this what you're facing?

**Comment by niallthompson** [comment]
Yes that is the same issue I am facing. You can only have one modal per page. Is there a workaround?

**Comment by AxelBriche** [comment]
Hello, kill modal content and reload modal with new content by ajax ?
